fix for failing compilation with GCC 10.2.0 (which now defaults to -fno-common)

  multiple definition of 'ocdebug' (in ocdebug.c and ocprint.c)

see https://github.com/Unidata/netcdf-c/issues/1725

--- netcdf-c-4.7.4/ncdump/ocprint.c.orig	2020-11-06 14:43:23.334448698 +0100
+++ netcdf-c-4.7.4/ncdump/ocprint.c	2020-11-06 14:43:34.208323588 +0100
@@ -56,7 +56,7 @@
 /*Mnemonic*/
 #define TOPLEVEL 1
 
-int ocdebug;
+extern int ocdebug;
 
 static OCerror ocstat;
 static OClink glink;
